src/audio/audio.c

```diff
@@ -1714,7 +1714,8 @@ void audio_init1()
 			}
 		}
 		m++;
-	} while(m + 1 < 0x2E);
+	// 46 is the number of files in ride_music_info_list
+	} while(m + 1 < 46);
```

Comment:
Refactor to a for loop. Is ride_music_info_list implemented if it is could use the macro to get the length of the array to get rid of the magic number and note.